### PR TITLE
feat(debug): Allow blurring video background during calls [WPB-724]

### PR DESCRIPTION
**Pull request by aweiss-dev**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-724" title="WPB-724" target="_blank">StoryWPB-724</a>  [Web] Blurred background on calls
  </td></table>
  <br />
 

## Description

This is a first MVP for the blurred background. 
It enabled blurring the self background using a debug util `wire.debug.enableCameraBlur()`. 

Next iteration:
- allow pre-loading the segmenter's models ahead of time so that blurring feels instant
- experiment a little more with webGL and see if we can optimize general blurring performances (see https://github.com/wireapp/wire-webapp/pull/17512/files)

## Performances

Currently the performances are not quite there yet. The blurring is quite costly. 
This is a typical example of the computation of a single frame of the video feed

image

We can see that the segmentation is taking very little time (some 2-3ms) but blurring takes 20ms+. Since we are supposed to have a budget of 16ms, this is not really our target performance. 
It's also quite intense on the main thread and leaves very little resources for the user. Those issues will need to be tackled.

### Using webWorker to compute the blur:

- 👍  leaves the main thread rather free
- 👎  frame rate drops and video gets quite laggy

### Using webgl to compute blurring on the GPU

- no performance gain have been found yet (https://github.com/wireapp/wire-webapp/pull/17512/files#diff-6ecbcd2983f79d5985879c4aa68724d2f00b4297aab3868a654217d6f3757768) Will need more experimenting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,10 +50,12 @@
     "redux-logdown": "1.0.4",
     "redux-thunk": "3.1.0",
     "speakingurl": "14.0.1",
+    "stackblur-canvas": "^2.7.0",
```
Comment (aweiss-dev) [inline package.json]: this can go

File: assets/mediapipe-models/selfie_multiclass_256x256.tflite
Comment (aweiss-dev) [inline assets/mediapipe-models/selfie_multiclass_256x256.tflite:1]: this can go

File: .husky/post-checkout
Comment (aweiss-dev) [inline .husky/post-checkout:1]: no need for LFS without the large file

File: .husky/post-commit
Comment (aweiss-dev) [inline .husky/post-commit:1]: no need for LFS without the large file

File: .husky/post-merge
Comment (aweiss-dev) [inline .husky/post-merge:1]: no need for LFS without the large file

File: .husky/pre-push
Comment (aweiss-dev) [inline .husky/pre-push:1]: no need for LFS without the large file

File: .gitattributes [hunk 2]
```diff
@@ -13,3 +11,4 @@
 *.ttf binary
 *.woff binary
 *.woff2 binary
+*.tflite filter=lfs diff=lfs merge=lfs -text
```
Comment (aweiss-dev) [inline .gitattributes]: no need for LFS without the large file

**Review by PatrykBuniX — APPROVED** [review]

**Pull request merged**